src/components/Resume/Resume.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { ProjectResume } from './ProjectResume/ProjectResume'
+import './Resume.module.scss'
+import AboutResume from './AboutResume/AboutResume'
+import { ExperienceResume } from './ExperienceResume/ExperienceResume'
+
+interface ResumeProps {
+  userId: string
+  /*
+  add other props here
+  */
+}
+
+//TODO: onSubmit whole Resume
```

Comment:
can this be implemented, or does it require auth/userId?

Reply:
it does require auth/userId. The whole create online resume does but I think it can merge along with the Project and Experience component.